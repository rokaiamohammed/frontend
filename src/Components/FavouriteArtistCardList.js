import React from 'react';
import Card from './FavouriteArtistCard';
import './FavouriteArtist.css';
import { Artists } from '../Components/Artists';

const FavouriteArtistCardList =({list}) =>{
    return(
        <div >
            { 
                list.map((user,i) =>{
                    return (
                        <Card 
                            key={i}
                            id={list[i]._id} 
                            image={Artists[7].image}
                            name={list[i].nickname}
                        />
                    );
                })
            }
        </div>
    )
}

export default FavouriteArtistCardList