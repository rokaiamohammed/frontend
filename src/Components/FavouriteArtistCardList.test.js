import React from 'react';
import { shallow, mount } from 'enzyme';
import FavouriteArtistCardList from './FavouriteArtistCardList';

const favouriteArtistCardList = [{
    key:2,
    id:2,
    image:'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQXbP65Mm3fFC5EE80LIRNsvWd-bStt_HftBS015aagdmwqygG6&usqp=CAU',
    name: 'Top hits'

  }]

it('FavouriteArtistCardList renders without crashing', () => {
  expect(shallow(<FavouriteArtistCardList list={favouriteArtistCardList} />)).toMatchSnapshot();
});