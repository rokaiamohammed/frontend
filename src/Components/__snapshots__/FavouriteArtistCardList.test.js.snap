// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavouriteArtistCardList renders without crashing 1`] = `ShallowWrapper {}`;
